src/jsonv/value.cpp

```diff
@@ -503,10 +503,11 @@ bool value::empty() const
         return _data.array->empty();
     case jsonv::kind::string:
         return _data.string->_string.empty();
+    case jsonv::kind::null:
+        return true; // by definition a null value is empty
     case jsonv::kind::integer:
     case jsonv::kind::decimal:
     case jsonv::kind::boolean:
-    case jsonv::kind::null:
     default:
         // Should never hit this...
```

Comment:
Just a note, your changes will make this comment inaccurate, since the `false` right below will be proper.